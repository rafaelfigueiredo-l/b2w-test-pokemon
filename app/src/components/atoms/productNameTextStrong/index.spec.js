import ProductNameText from "./";

describe("<ProductNameText />", () => {
  const component = <ProductNameText>Texto</ProductNameText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ProductNameText component", () => {
    expect(wrapper.find(".productNameText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ProductNameText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".productNameText").children()).toHaveLength(1);
  });
});
