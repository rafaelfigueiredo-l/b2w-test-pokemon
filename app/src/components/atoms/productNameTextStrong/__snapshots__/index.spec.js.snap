// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductNameText /> should mount correctly ProductNameText component 1`] = `
<styled.span
  className="productNameText"
>
  <span
    className="sc-AxjAm dOojyz productNameText"
  >
    Texto
  </span>
</styled.span>
`;
